common/Timer.cc

```diff
@@ -42,21 +42,35 @@ void Timer::cancel() {
     this->canceled = true;
 }
 
-Timer Timer::fork(ConstExprStr name) {
+Timer Timer::clone(ConstExprStr name) {
     Timer forked(log, name, prev, {}, start);
     forked.args = args;
+    forked.tags = tags;
     forked.canceled = canceled;
     return forked;
 }
 
+void Timer::setTag(ConstExprStr name, ConstExprStr value) {
```

Comment:
This is the ~first time Timers are mutated.
Can we make this be a construction-time-only thing?